<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>907</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>921</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="UPLODEVIDEO">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>817</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>UPLODE VIDEO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BACKTOHOME">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>820</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>BACK TO HOME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AI_BUTTON">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>750</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>AI</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="statues">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>760</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser {
    background: transparent;
	border: none;
}</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
